packages/api/src/utils/highlightGenerator.ts

```diff
@@ -419,7 +422,7 @@ export function makeHighlightNodeAttributes(
           isParagraphStart && !i && quote && (quote += '\n')
           quote += text
         }
-
+        console.log('quote', quote)
```

Comment:
not sure if you need this anymore

Reply:
haha, i'm removing this and then i saw your comment @jacksonh